src/main.js

```diff
@@ -92,6 +95,8 @@ function compileTo(src, dest) {
 
 if (argv.interactive) {
     repl.start();
+} else if (argv._.length === 1) {
+    compileTo(argv._[0], undefined);
```

Comment:
Just call it `compileHelper` or something.

On Tue, Nov 17, 2015 at 4:09 PM Noah Morrison <notifications@github.com>
wrote:

> In src/main.js
> <https://github.com/wavebeem/squiggle/pull/113#discussion_r45142271>:
>
> > @@ -92,6 +95,8 @@ function compileTo(src, dest) {
> >
> >  if (argv.interactive) {
> >      repl.start();
> > +} else if (argv._.length === 1) {
> > +    compileTo(argv._[0], undefined);
>
> There's already a function called compile in compile.js, any other naming
> suggestions?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wavebeem/squiggle/pull/113/files#r45142271>.
>